resources/analysisTools/snvPipeline/snvFilter.sh

```diff
@@ -1,14 +1,335 @@
 #!/bin/bash
```

Comment:
The original snvFilter.sh was actually just dead code that was never executed. Your code instead is based on filter_vcf.sh, right?

The somehow cleanest way to proceed would be to delete snvFilter.sh altogether and put your changes into filter_vcf.sh. This is also the easiest way that get github to show a useful diff. On the long run snvFilter.sh should be deleted as file and from the plugin XML (unless @GWarsow sees any problem in this, e.g. because it is used in later plugin versions).

As it is now, I need to do the diff externally, and add my comments here, which is much more work. So, for now I am not reviewing this code.